modules/users/client/views/settings/manage-social-accounts.client.view.html

```diff
@@ -8,8 +8,11 @@ <h3 class="col-md-12 text-center" ng-show="vm.hasConnectedAdditionalSocialAccoun
     </div>
     <div ng-repeat="(providerName, providerData) in vm.user.additionalProvidersData" class="social-account-container">
       <img ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
```

Comment:
Good call, there's several other places in the project where it would make sense too that we'll have to get round to eventually.